translation from c00/30/16 to 20 (#66)
IllusionMods/HS2-Translation: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/30/15/4/translation.txt

//
// Dumped for Honey Select 2 v1.1.3 by Text Dump v1.4.1
//

> もしかして、お疲れで早く休まれたかったのでしょうか？　配慮が足りず申し訳ございません。=Could it be that you were tired and wanted to go to bed early? I appologise for being inconsiderate.

Reviewer: ```suggestion
もしかして、お疲れで早く休まれたかったのでしょうか？　配慮が足りず申し訳ございません。=Maybe you were tired and wanted to go to bed early? Sorry for being so inconsiderate.
```
